pylsp/__init__.py

```diff
@@ -3,12 +3,27 @@
 
 import os
 import pluggy
+from . import _version
 from ._version import __version__
 
+def convertVersionInfo(version: str) -> (int, ..., str):
```

Comment:
```suggestion
def convert_version_info(version: str) -> (int, ..., str):
```

We don't use camel case names for functions.

Reply:
Fixed.